src/app/main/component/ubs/components/ubs-order-details/ubs-order-details.component.html

```diff
@@ -178,6 +178,9 @@ <h5>{{ 'order-details.question-certificate' | translate }}</h5>
           <div class="points">
             <h5>{{ 'order-details.bonus-need' | translate }}</h5>
             <p>{{ 'order-details.bonus-left' | translate }} {{ certificateLeft | localizedCurrency }}</p>
+            <p>
+              <a href="" class="bonus-how-to-link">{{ 'order-details.bonus-how-to-get' | translate }}</a>
```

Comment:
what is the point of adding a non-working link?

Reply:
we dont have a correct link at the moment